controller/deployments.go

```diff
@@ -192,6 +193,11 @@ func (c *DeploymentsController) getKubeClient(ctx context.Context) (kubernetes.K
 	return kc, nil
 }
 
+func httpStatusFailed(status int) bool {
```

Comment:
No no, this is not good. A redirect would not fall in the boundary of status codes and might still be correct. Please just replace the `200` with `http.StatusOK` and so forth.